Components/Lists/list.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lists</title>
+    <link rel="stylesheet" href="../../Docs/doc.css">
+    <link rel="stylesheet" href="list.css">
+</head>
+<body>
+    <nav class='nav'>
+        <div >
+            <a class="nav-header" href="">Ozone UI</a>
```

Comment:
Suggestion: You can provide a placeholder # in href instead of leaving it blank. Or if It is to re-direct on the home page then provide /

Reply:
sure, will add that